src/neon/utf8check.rs

```diff
@@ -178,7 +178,7 @@ fn check_overlong(
     }
 }
 
-pub struct ProcessedUtfBytes {
+pub(crate) struct ProcessedUtfBytes {
```

Comment:
excellent call on locking this down to crate only :)

Reply:
just copying the other module which was different! 😂